res/css/views/rooms/_EventBubbleTile.scss

```diff
@@ -89,8 +92,8 @@ limitations under the License.
         position: absolute;
         top: -1px;
         bottom: -1px;
-        left: -60px;
```

Comment:
This should not be -60px, instead it should be 49px, the same value as the inline start margin of the EventTile.